jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -303,7 +304,7 @@
                                     (send self :get-proximity l/r :left))
                                  (send self :interpolatingp))
                        (unix::usleep 1000))))
-                 (send self :cancel-move-hand)
+                 (send self :cancel-move-hand l/r)
                  (send self :cancel-angle-vector)
```

Comment:
so, it means that is we do `(jsk_arc2017_baxter::arc-init :ctype :default-controller)`, `:cancel-vector` cancels all the trajectory?